src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/AbstractCaseView.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.ccd.domain.model.aggregated;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+public abstract class AbstractCaseView {
+
+    @JsonProperty("case_id")
+    private String caseId;
+    @JsonProperty("case_type")
+    private CaseViewType caseType;
+    private CaseViewTab[] tabs;
```

Comment:
Giri do we need to return and show tabs in a case history? the story only mentions displaying 'old case fields data'